### ISSUE TITLE
Implement Conflict Check

### ISSUE THREAD

**Issue by alecxvs** [umkc-cs-451-2020-spring/semester-project-group-10#15]
Implement the conflict check system based on #11. This will probably depend on #12 to be complete.

**Comment by alecxvs** [comment]
This seems a bit vague, we may be able to split this into more tasks if this is too complex. Thoughts? @BigWendl
